shaders/color_invert.glsl

```diff
@@ -7,10 +7,14 @@ layout(location = 0) out vec4 fragColor;
 layout(location = 1) uniform sampler2D inputImageTexture;
 layout(location = 0) uniform vec2 screenSize;
 
+vec4 processColor(vec4 sourceColor){
+    return vec4((1.0 - sourceColor.rgb), sourceColor.w);
+}
+
 void main()
 {
     vec2 textureCoordinate = gl_FragCoord.xy / screenSize;
-    lowp vec4 textureColor = texture(inputImageTexture, textureCoordinate);
+    lowp vec4 sourceColor = texture(inputImageTexture, textureCoordinate);
```

Comment:
Rename textureColor inside `processColor` in all other place name should be as before

Reply:
Added renamed textureColor inside processColor in all other place name should be as before